libr/core/core.c

```diff
@@ -767,13 +767,24 @@ static int autocompleteProcessPath(RLine *line, const char *path, int argv_idx)
 
 	list= r_sys_dir (dirname);
 	n = strlen (basename);
+	char *tmpstring;
+	int chgdir = (!strncmp (line->buffer.data, "cd ", 3) ? 1 : 0);
```

Comment:
this is bool, not int, no need to do the ternary   operator. bool can be only 1 or 0 and its handled by the compiler